LEAF_Request_Portal/sources/Workflow.php

```diff
@@ -1549,7 +1567,22 @@ public function getDescription($workflowID){
 
     public function getHistory($filterById)
```

Comment:
Let's get this function set up with hints and type returns.

Reply:
updated